angular-konami.js

```diff
@@ -17,7 +20,7 @@ angular.module('konami', [])
         }
 
         // Let user define a custom code.
-        var konamiKeys = attr.konamiKeys || konamiKeysDefault;
+        var konamiKeys = konamiKeys || konamiKeysDefault;
```

Comment:
Hello, @xavisavvy! I don't really understand what have you modified here. It just seem to me that this code would break with an undefined exception. Shame on me not to have tests for this project, but isn't that right?